libs/api/utils/src/core/api/parseApiKey.test.ts

```diff
@@ -0,0 +1,12 @@
+import { describe, expect, it } from 'bun:test'
+import { parseApiKey as func } from './parseApiKey'
```

Comment:
**suggestion:** (blocking) We shouldn't rename function as it makes tests less readable and it can be "any function" with this generic name.